imports/voting/startup/server/health-check.js

```diff
@@ -0,0 +1,29 @@
+import { WebApp } from 'meteor/webapp';
+
+import { onErc20ContractReceiving } from '/imports/ethereum';
+import { Proposals } from '../../api/models/proposals';
+
+onErc20ContractReceiving(erc20contract =>
+  WebApp.connectHandlers.use('/health', (req, res) => {
+    let web3Provider;
+    try {
+      const balance = erc20contract.balanceOf('0x0');
+      web3Provider = typeof balance === 'object' ? 'ok' : 'invalid-value';
+    } catch (e) {
+      web3Provider = 'exception';
+    }
+
+    let database;
+    try {
+      const count = Proposals.find().count();
+      database = (typeof count !== 'number' && 'invalid-value') ||
+        (count > 0 && 'ok') || 'no-proposals';
```

Comment:
I guess `database: no-proposals` should be not accepted, right?

Reply:
Yes, basically, all that is not "ok".